ui-kit/src/main/kotlin/org/p2p/uikit/components/UiKitAmountsView.kt

```diff
@@ -57,8 +57,8 @@ class UiKitAmountsView @JvmOverloads constructor(
 
         with(binding) {
             val originalTextSize = editTextTokenAmount.textSize
-            editTextTokenAmount.doOnTextChanged { text, _, _, _ ->
-                handleAmountTextChanged(editTextTokenAmount, textViewTokenAutoSizeHelper, text, originalTextSize)
+            AmountFractionTextWatcher.installOn(editTextTokenAmount, maxSymbolsAllowed = 2) { amount ->
```

Comment:
use constant please except of just "2"

Reply:
I would say it's more readable since we have named args here